examples/devsite-landing-demo/style.css

```diff
@@ -95,15 +95,11 @@ option {
   border-left: 15px solid #4db6ac !important;
 }
 
-#blockly-5 {
-  border-left: 15px solid #ffcdd2 !important;
-}
-
-#blockly-7 {
+#blockly-6 {
   border-left: 15px solid #ef9a9a !important;
 }
 
-#blockly-8 {
+#blockly-7 {
   border-left: 15px solid #d7ccc8 !important;
 }
```

Comment:
I don't think these CSS IDs are technically supported. We should be using a [custom toolbox](https://blocklycodelabs.dev/codelabs/custom-toolbox/index.html?index=..%2F..index#2), or [category CSS](https://developers.google.com/blockly/guides/configure/web/toolbox#category_css).

Reply:
I tried using category css and it works.

Should I update on this PR or open another one when this is done?

Reply:
You can make updates on this PR! Thanks for testing that out :D

Reply:
Hey @mobyw since this fixes the issue with the live demo we'll actually go ahead and merge it now so we can make sure the demo is running. If you're still up for fixing the css to use the category CSS, that would be great and we'd love a follow up PR! Thanks!